srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
    main()
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 99, in main
    main()
  File "run_linearization_heuristic.py", line 99, in main
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    main()
    main()
    delta = all_instances[instance_index][0]
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
IndexError: list index out of range
    delta = all_instances[instance_index][0]
IndexError: list index out of range
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
IndexError: list index out of range
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 871, in <module>
Traceback (most recent call last):
    delta = all_instances[instance_index][0]
Traceback (most recent call last):
    main()
  File "run_linearization_heuristic.py", line 871, in <module>
IndexError: list index out of range
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 99, in main
    main()
    main()
    delta = all_instances[instance_index][0]
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
IndexError: list index out of range
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: Terminating job step 9431456.14
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: Terminating job step 9431456.11
srun: Terminating job step 9431456.10
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: Terminating job step 9431456.1
srun: Terminating job step 9431456.13
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: Terminating job step 9431456.9
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: Terminating job step 9431456.7
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: Terminating job step 9431456.12
srun: Terminating job step 9431456.0
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: Terminating job step 9431456.2
srun: Terminating job step 9431456.8
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: Terminating job step 9431456.5
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: Terminating job step 9431456.3
srun: Terminating job step 9431456.6
srun: Terminating job step 9431456.4
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 871, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
    delta = all_instances[instance_index][0]
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
    main()
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
IndexError: list index out of range
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
    main()
    main()
    delta = all_instances[instance_index][0]
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
    main()
  File "run_linearization_heuristic.py", line 99, in main
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
IndexError: list index out of range
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
  File "run_linearization_heuristic.py", line 871, in <module>
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
    delta = all_instances[instance_index][0]
IndexError: list index out of range
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
    main()
IndexError: list index out of range
IndexError: list index out of range
IndexError: list index out of range
IndexError: list index out of range
IndexError: list index out of range
IndexError: list index out of range
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431456.24
srun: Terminating job step 9431456.26
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431456.23
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431456.28
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431456.29
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431456.20
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431456.25
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431456.27
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431456.18
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431456.21
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431456.22
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431456.17
srun: Terminating job step 9431456.16
srun: Terminating job step 9431456.19
srun: Terminating job step 9431456.15
